Packs/FireEyeETP/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Playbooks
+##### New: FireEye ETP - Indicators Hunting
+Created a new playbook for threat hunting and detection of IOCs such as domains, IP addresses, sender and recipient email addresses. 
+<!--Note that multiple search values should be separated by commas only (without spaces or any special characters). (Available from Cortex XSOAR 6.5.0).-->
```

Comment:
@ShirleyDenkberg This note format isn't viewable. Is it on purpose?

Reply:
I excluded it. 
The reason I excluded this sentence is that it repeats itself in every playbook description, and I thought it irrelevant to the RN.